mandatory/4-tax.js

```diff
@@ -17,8 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
-
+function addTaxAndFormatCurrency(price) {
+  return (price+(price*20/100)).toFixed(2);
```

Comment:
good expression, you could declare some lets to make the code easier to understand

Reply:
Thank you for your feedback, Luan.